### PR TITLE
Feature/page layouts

### PR DESCRIPTION
**Pull request by wfercanas**

### CODE REVIEW

File: src/containers/App.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import { GlobalStyle } from "../components/styles/globals";
+import SearchResults from '../pages/SearchResults';
 
 function App() {
   return (
     <div className="App">
       <GlobalStyle />
-      <h1>Music App</h1>
+      <SearchResults></SearchResults>
```
Comment (alfredomtzg) [inline src/containers/App.js]: Return <h1> Music App</h1>   and see the video from Oscar

**Review by alfredomtzg — CHANGES_REQUESTED** [review]
I think everything is fine, I just can't do the merge until you can see Oscar's video or I can do it but you may change the way you see it and you have to make changes again

**Comment by wfercanas** [comment]
I think its functional by now, so I prefer that you do the merge and then we'll see how we can improve it.

File: src/containers/App.js (hunk 1 above)
Comment (wfercanas) [reply src/containers/App.js]: Below you can see my fix to undo changes in App.js

**Pull request merged**